site/content/in-dev/unreleased/admin-tool.md

```diff
@@ -74,14 +74,7 @@ The Polaris Admin Tool must be executed with the same configuration as the Polar
 configuration can be done via environment variables or system properties.
 
 At a minimum, it is necessary to configure the Polaris Admin Tool to connect to the same database
-used by the Polaris server. This can be done by setting the following system properties:
-
-```shell
-java \
-    -Dpolaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml \
-    -Dpolaris.persistence.eclipselink.persistence-unit=polaris \
-    -jar quarkus/admin/build/polaris-quarkus-admin-*-runner.jar
-```
+used by the Polaris server.
```

Comment:
Do users need to do any action here to make this happen? If not, would it be better to remove this line altogether?

Reply:
yes, but the immediate line below re-directs the user to move to metastore.md page to configure the metastore, so this was kind a redundant. Hence removed.